conda_smithy/configure_feedstock.py

```diff
@@ -1385,7 +1385,9 @@ def _load_forge_config(forge_dir, exclusive_config_file):
         },
         "recipe_dir": "recipe",
         "skip_render": [],
-        "bot": {"automerge": False,},
+        "bot": {"automerge": False},
+        "conda_forge_output_validation": False,
+        "conda_forge_ci_setup_version": "2",
```

Comment:
This is too much customization. Let's just change 2 to 3

Reply:
done.